tests/transformations/subtests/transpile-js/webpack/webpack.config.js

```diff
@@ -13,6 +13,13 @@
 module.exports = {
   module: {
     rules: [
+      {
+        // TODO workaround for broken @babel/runtime/helpers/esm
+        include: /@babel.runtime.helpers.esm/,
+        resolve: {
+          fullySpecified: false,
+        },
+      },
```

Comment:
@sokra I believe this should be fixed if this test is updated to `@babel/*@7.12.x` - the `@babel/runtime` package now includes Package Exports that map the extensionless bare specifiers to `.js` extensions, so it should be spec-compliant.

```suggestion
```